app/safe/internal/server/route/fetch/fetch.go

```diff
@@ -93,7 +93,8 @@ func Fetch(
 
 	secret, err := collection.ReadSecret(cid, workloadId)
 	if err != nil {
-		log.WarnLn(&cid, "Fetch: Problem reading secret", err.Error())
+		log.WarnLn(&cid, "Fetch: Attemted to read secret from disk.")
+		log.TraceLn(&cid, "Likely expected error. No need to panic:", err.Error())
```

Comment:
This message looked like a severe error and was confusing the developer when debugging the system behavior.

Renamed it to make it look intentionally less alarming.